test/testUtils/mayaUtils.py

```diff
@@ -232,7 +232,12 @@ def previewReleaseVersion():
         return 98
     elif majorVersion == 2020:
         return 110
+    elif majorVersion == 2022:
+        return 122
 
     match = prRe.match(cmds.about(v=True))
 
     return int(match.group(1)) if match else sys.maxsize
+
+def mayaMajorVersion():
+    return int(cmds.about(majorVersion=True))
```

Comment:
Trivial - but new helper to return Maya major version. So all checks can go thru mayaUtils/ufeUtils.